package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "motrix-webextension",
+  "version": "2.0.0",
+  "description": "WebExtension for Motrix download manager",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config=webpack.config.js --mode=production",
+    "start:firefox": "npm run build && web-ext run --firefox=firefoxdeveloperedition --source-dir ./dist/",
+    "package": "npm run build && node package.js chrome firefox"
```

Comment:
@gautamkrishnar The package script seems to throw an error but executes perfectly. So ignore the error while executing it. I think most of the things are in place to test and publish the extension for chrome, edge and firefox.